Octokit/Models/Response/EmailAddress.cs

```diff
@@ -25,24 +27,54 @@ public EmailAddress(string email, bool verified, bool primary)
         public string Email { get; protected set; }
 
         /// <summary>
-        /// true if the email is verified; otherwise false
+        /// True if the email is verified; otherwise false
         /// </summary>
         public bool Verified { get; protected set; }
 
         /// <summary>
-        /// true if this is the users primary email; otherwise false
+        /// True if this is the users primary email; otherwise false
         /// </summary>
         public bool Primary { get; protected set; }
 
+        /// <summary>
+        /// "private" or "public" if the email address is the primary;
+        /// otherwise null
+        /// <summary>
+        public StringEnum<EmailVisibility> Visibility { get; protected set; }
```

Comment:
The API response actually has a `null` value rather than the string "null" so what we need to do is make this property nullable (eg `StringEnum<EmailVisibility>? Visibility`)

Reply:
(and note that this will not deserialize currently, but once #1760 is in, this should then work!)